app/src/main/java/com/framgia/soundclound/data/model/Track.java

```diff
@@ -59,6 +57,8 @@
     @Expose
     private String mDisplayDate;
 
+    private boolean isChecked;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isChecked has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName